src/app/(content-pages)/methodology/layout.tsx

```diff
@@ -1,6 +1,15 @@
-export const metadata = {
-  title: "Methodology",
-};
+import { generateMetadata } from "@/utilities/generateMetaData";
+
+const title = "Methodology";
+const description =
+  "Clean & Green Philly combines several public datasets in order to categorize Philadelphia’s vacant properties based on how important it is that someone intervene there."; //??
```

Comment:
Remove `//??` comment. Also, let's remove "in order" from the description to shorten it.